package.json

```diff
@@ -2,41 +2,56 @@
   "name": "semantic-release-monorepo",
   "version": "0.0.0-development",
   "description": "Plugins for `semantic-release` allowing it to be used with a monorepo.",
+  "type": "module",
   "main": "src/index.js",
+  "publishConfig": {
+    "access": "public"
+  },
   "files": [
     "src"
   ],
   "scripts": {
-    "format": "prettier --write --single-quote --trailing-comma es5",
-    "format:all": "yarn format \"./**/*.js\"",
+    "lint": "eslint --ignore-path .eslintignore --ext .js,.ts,.cjs .",
+    "lint:fix": "npm run lint -- --fix",
+    "prettier": "prettier \"**/*.{ts,js,jsx,css,cjs,json}\"",
+    "prettier:fix": "prettier \"**/*.{ts,js,jsx,css,cjs,json}\" --write",
     "test": "jest"
   },
   "license": "MIT",
   "peerDependencies": {
     "semantic-release": ">=15.11.x"
   },
   "dependencies": {
+    "semantic-release-plugin-decorators": "^4.0.0",
```

Comment:
Was hoping to use this branch as a workaround until this PR is merged, but this doesn't seem to resolve to anything? Latest looks like it's still v3.0.1

Reply:
I'm waiting for integrate theses changes... 
https://github.com/pmowrer/semantic-release-plugin-decorators/pull/31